build/DependencyVersions.props

```diff
@@ -13,7 +13,7 @@
     <NuGetVersion>4.5.0-preview2-4529</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25708-01</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25708-01</NETStandardLibraryNETFrameworkVersion> <!-- Once we have a new version of this package available, this version will have to be updated -->
```

Comment:
This has to change in the cli repo as well. This only controls what the sdk uses to test itself

Reply:
Noted, I'll make sure it changes in the CLI repo as well.